ffw/UIRPC.js

```diff
@@ -391,6 +391,12 @@ FFW.UI = FFW.RPCObserver.create(
                 SDL.ResetTimeoutPopUp.resetTimeoutRPCs.removeObject('VR.PerformInteraction');
               }
               array['UI.PerformInteraction'] =  request.params.timeout/1000 * increase_value;
+
+              if (request.params && request.params.choiceSet == null) {
```

Comment:
@LitvinenkoIra are you sure it is a proper condition to remove the UI part?

Reply:
@AKalinich-Luxoft Here we need to check that Performinteraction from mobile was sent with VR_ONLY mode. So according to this diagram https://luxproject.luxoft.com/confluence/display/APPLINK/PerformInteraction#PerformInteraction-1.15.2.RequestsUI/VR-portionsprocessing I made an assumption that UI should be send without choiceSet param in this case